src/form/Form.jsx

```diff
@@ -1212,6 +1217,20 @@ function Form({
     } else handleRedirect({ metadata });
   };
 
+  const elementOnView = onView
+    ? (elementId, isVisible) => {
+        if (typeof onView === 'function') {
```

Comment:
i think this check can be the top-level ternary so you don't need an additional check here